apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/TransactionMode.cs

```diff
@@ -15,14 +15,23 @@
 namespace Google.Cloud.Spanner.Data
 {
     /// <summary>
+    /// Indicates the mode for the transaction.
     /// </summary>
     public enum TransactionMode
     {
         /// <summary>
+        /// Read-only transactions provide guaranteed consistency across several reads,
+        /// but do not allow writes. Read-only transactions can be configured to read at
+        /// timestamps in the past. Read-only transactions do not need to be committed and
+        /// do not take locks.
         /// </summary>
         ReadOnly = 0,
 
         /// <summary>
+        /// Locking read-write transactions are the only transaction type that support writing
+        /// data into Cloud Spanner. These transactions rely on pessimistic locking and, if
```

Comment:
If you read a few external docs on spanner, you may very well be led to believe that they do support it -- but they dont.  The spanner folks really need to improve their documentation.  When they talk about using 2-phase commit, they are talking about their internal implementation in how they handle spanner across nodes.  They don't expose two phase commit externally.